Formula/xfig.rb

```diff
@@ -0,0 +1,57 @@
+class Xfig < Formula
+  desc "Facility for interactive generation of figures"
+  homepage "https://mcj.sourceforge.io"
+  url "https://downloads.sourceforge.net/mcj/xfig-3.2.8a.tar.xz"
+  sha256 "ba43c0ea85b230d3efa5a951a3239e206d0b033d044c590a56208f875f888578"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    regex(%r{url=.*?/xfig[._-]v?(\d+(?:\.\d+)+[a-z]?)\.t}i)
+  end
+
+  depends_on "fig2dev"
+  depends_on "ghostscript"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "libx11"
+  depends_on "libxaw3d"
+  depends_on "libxi"
+  depends_on "libxpm"
+  depends_on "libxt"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --with-appdefaultdir=#{etc}/X11/app-defaults
+    ]
+
+    system "./configure", *args
+    # "LDFLAGS" argument can be deleted in the next release. See discussion at
+    # https://sourceforge.net/p/mcj/discussion/general/thread/36ff8854e8/#fa9d.
+    system "make", "LDFLAGS=-ltiff -ljpeg -lpng", "install-strip"
+  end
+
+  test do
+    if ENV["HOMEBREW_GITHUB_ACTIONS"]
+      assert_equal "Xfig #{version}", shell_output("#{bin}/xfig -V 2>&1").strip
+    else
+      # Write a version 1.3 fig file and update it, i.e., convert it to the
+      # newest version. Just check whether the old file was backed up.
+      # Converting a fig file is the only action that can be done without a GUI.
+      (testpath/"old.fig").write <<~EOS
+        1200 1 800 400
+        1 1 1 15 20. 1 400 200 180 90 400 20 400 380
+        2 6 1 15 10. 0 0 10 10 520 50 720 50 9999 9999
+        3 10 1 15 10. 0 0 10 10 320 550 520 750 620 450 720 550 9999 9999
+        4 0 0 0 12 200 600 700 Failed miserably
+        5 3 1 15 5. 0 0 0 10 10 0 0 200 0 144 144 0 200
+      EOS
+      system "#{bin}/xfig", "-update", "old.fig"
+      assert_predicate testpath/"old.fig.bak", :exist?
+    end
```

Comment:
```suggestion
    assert_equal "Xfig #{version}", shell_output("#{bin}/xfig -V 2>&1").strip
```

Test blocks exist almost exclusively for GitHub Actions. No point or benefit in having behaviour deviate between the two.